.pylintrc

```diff
@@ -62,6 +62,7 @@ ignore-imports=yes
 #    too-many-statements
 # and fix those issues.
 
+disable=no-self-use,broad-except,too-many-statements,too-many-locals,too-many-branches,dangerous-default-value,not-context-manager,duplicate-code
```

Comment:
- no-self-use actually means that the method does not use `self` within it, and should be pulled out of the class/instance into a standalone function. Or perhaps you can define it as `@staticmethod`, but we don't use that a lot.
- too-many-statements: how about pulling out some parts of the main() method into separate functions? It indicates that the method is too long/sprawling/complex.
- too-many-locals: might be addressed in same way as too-many-statements?

For duplicate-code: OK, I'm fine with disabling this one.